api/app/tests/apps/email_integrations/test_mail.py

```diff
@@ -122,24 +148,50 @@ def test_send_mail_reporter_created(self):
         # we want a history entry when a email was sent
         self.assertEqual(Note.objects.count(), 1)
 
+    def test_send_mail_reporter_created_text_contains_url(self):
+        """
+        URL's in the signal text are removed from the context and do not end up in the email
+        """
+        self.signal.text = 'These links should be removed: https://www.example.com/just/a/path/ and http://test.com'
```

Comment:
See comment about internationalized domains or FTP links, the current link finding is not enough. I would also test the `url_pattern` regular expression in isolation with a list of pathalogical cases.

Reply:
Done